client/cypress/integration/query/parameter_spec.js

```diff
@@ -129,14 +129,14 @@ describe('Parameter', () => {
         .find('.ant-select')
         .click();
 
-      cy.contains('li.ant-select-dropdown-menu-item', 'value1')
+      cy.contains('li.ant-select-dropdown-menu-item', 'value2')
```

Comment:
Needed to change to `value2` since now I make sure the first dropdown option is selected when value is invalid or empty.